dev/generate_bom.py

```diff
@@ -83,6 +85,8 @@ def init_argument_parser(cls, parser):
                         help="Docker registry to push the container images to.")
     parser.add_argument('--changelog_output', default='',
                         help="Output file to write the changelog to.")
+    parser.add_argument('--nightly', default=False, action='store_true',
```

Comment:
Ok makes sense. For now, I'm going to add a flag to alias the 'real' BOM as `nightly.yml` at the time of creation so we can support that for 1.0. I like the after the fact aliasing, but I actually think that should go into `dev/promote_bom.py` since we're doing something similar there when we want to release.